eskip/eskip.go

```diff
@@ -32,16 +32,18 @@ var (
 //        r0: Source("127.0.0.1/8", "10.0.0.0/8") -> inlineContent("OK") -> <shunt>
 //        # actual route
 //        edit_r0: ClientIP("127.0.0.1/8", "10.0.0.0/8") -> inlineContent("OK") -> <shunt>
-func NewEditor(reg *regexp.Regexp, repl string) *Editor {
+func NewEditor(reg *regexp.Regexp, repl string, sep string) *Editor {
 	return &Editor{
 		reg:  reg,
 		repl: repl,
+		sep:  sep,
```

Comment:
Why do we need to know separator here?

Reply:
I don't see any obvious reason for separator here too... However, won't we need it for some debug/logging/etc purposes?

Reply:
The Editor does not have to know that there is any separator involved at all - it should only know about match and replacement values